resources/lib/entrypoint/service.py

```diff
@@ -180,7 +180,8 @@ def check_update(self, forced=False):
         '''
         LOG.info("--[ check updates/%s ]", objects.version)
         kodi = "DEV" if settings('devMode.bool') else xbmc.getInfoLabel('System.BuildVersion')
-
+        # FIXME we do not want their updates
```

Comment:
I think we may want to point it to some our metadata instead of theirs...

Reply:
I don't fully understand the meaning of it. They maintain a separate branch for "objects", but the objects are also in master, so it's some weird dynamic automatic update functionality.

Is that the Kodi way or is this just overly complex for the sake of it?

Reply:
Can we drop this altogether?.. for me this looks like some overcomplication (though I don't know Kodi well enough to judge that).

Reply:
I did some research and it doesn't seem like it's the proper way to do automatic updates. If automatic updates are enabled, any new version published to a repo of our choice will be installed.

I'll be removing this code.